Site1.Master.cs

```diff
@@ -15,35 +15,35 @@ protected void Page_Load(object sender, EventArgs e)
             {
                 if (Session["role"].Equals(""))
                 {
-                    LinkButton1.Visible = true; //user login link button
-                    LinkButton2.Visible = true; //sing up link button
+                    UserLogin.Visible = true; 
+                    SignUp.Visible = true; 
 
-                    LinkButton3.Visible = false; //logout link button
-                    LinkButton7.Visible = false; //hello user login link button
+                    LogOut.Visible = false;
+                    UserNameDisplay.Visible = false;
 
-                    LinkButton6.Visible = true; //admin login link button
+                    AdminLogin.Visible = true; 
```

Comment:
### good enough. 

I know what you're referring to now, so that's good. But I don't know what element you're trying to disable.